### PR TITLE
Consistently use Go module "require" for testing

### PR DESCRIPTION
**Pull request by ErlendHaa**
When we introduced the module to the repository for the first time, we didn't go back and update old tests. Hence we have a good mix of manually comparisons and the use of require. 

To keep life simple for us, I've also removed all uses of the module assert.

### CODE REVIEW

File: cmd/query/main_test.go [hunk 1]
```diff
@@ -619,12 +618,12 @@ func TestLogHasNoSas(t *testing.T) {
 
 		requireStatus(t, testcase, w)
```
Comment (achaikou) [inline cmd/query/main_test.go:620]: Maybe also update utils_main_test.go? It has some "if err != nil"

File: internal/vds/vds_test.go [hunk 2]
```diff
@@ -83,15 +82,11 @@ func TestMetadata(t *testing.T) {
...
+	require.NoErrorf(t, err, "Failed to retrive metadata, err %v", err)
 
 	var meta Metadata
 	err = json.Unmarshal(buf, &meta)
-	if err != nil {
-		t.Fatalf("Failed to unmarshall response, err: %v", err)
-	}
+	require.NoErrorf(t, err, "Failed to unmarshall response, err: %v", err)
 
 	if !reflect.DeepEqual(meta, expected) {
```
Comment (achaikou) [inline internal/vds/vds_test.go:91]: DeepEqual probably can go too?

File: internal/vds/vds_test.go [hunk 3]
```diff
@@ -635,16 +498,10 @@ func TestInvalidFence(t *testing.T) {
...
-		if err.Error() != expected {
-			msg := "Unexpected error message, expected \"%s\", was \"%s\""
-			t.Errorf(msg, expected, err.Error())
-		}
-	}
+	require.ErrorContainsf(t, err,
+		"invalid coordinate [1 1 0] at position 1, expected [x y] pair",
+		"Incorrect error msg",
+	)
 }
```
Comment (achaikou) [inline internal/vds/vds_test.go:506]: To "TestFenceInterpolation"
exchange a lot with require.NotEqual ?

File: internal/vds/vds_test.go [hunk 4]
```diff
@@ -689,19 +546,16 @@ func TestFenceInterpolationDefaultIsNearest(t *testing.T) {
...
```
Comment (achaikou) [inline internal/vds/vds_test.go:547]: The one above is fine, if you don't want to change it, but not sure if you wanted to but didn't go through the whole file.

File: internal/vds/vds_test.go [hunk 5]
```diff
@@ -767,9 +614,9 @@ func TestSurfaceUnalignedWithSeismic(t *testing.T) {
 		interpolationMethod,
 	)
 	require.Len(t, buf, len(targetAttributes), "Wrong number of attributes")
-	require.NoError(t, err, "Failed to fetch horizon")
+	require.NoErrorf(t, err, "Failed to fetch horizon")
 	result, err := toFloat32(buf[0])
-	require.NoError(t, err, "Failed to covert to float32 buffer")
+	require.NoErrorf(t, err, "Failed to covert to float32 buffer")
 	require.Equalf(t, expected, *result, "Horizon not as expected")
 }
```
Comment (achaikou) [inline internal/vds/vds_test.go:622]: Update "TestSurfaceWindowVerticalBounds" too?

Comment (ErlendHaa) [reply internal/vds/vds_test.go:622]: I don't want to use `require.Equal(t, err, testcase.Err)` here if that's what you mean. Then I have to hardcode the exact error message into each test case

File: internal/vds/vds_test.go [hunk 6]
```diff
@@ -573,7 +572,7 @@ func TestOnly3DSupported(t *testing.T) {
 		handle.Close()
 	}
 
-	require.ErrorContainsf(t, err, "3 dimensions, got 4", "Incorrect error msg")
```
Comment (achaikou) [inline internal/vds/vds_test.go]: why did you remove "got 4"?

**Review by achaikou — COMMENTED** [review]
Get ready for merge.
And see if you want to fix some of the remaining results when searching for t.Error and t.Fatal

File: internal/vds/vds_test.go (hunk 6 above)
Comment (ErlendHaa) [reply internal/vds/vds_test.go]: Because my vim macro was incorrect 😄 I'll fix it on squash

**Review by achaikou — APPROVED** [review]
